shared/actions/chat2/index.js

```diff
@@ -1193,6 +1193,7 @@ const startConversation = (action: Chat2Gen.StartConversationPayload, state: Typ
   // we handled participants or tlfs
   if (participants) {
     users = participants
+    conversationIDKey = Constants.findConversationFromParticipants(state, I.Set(users))
```

Comment:
This is a separate change to the rest of the PR, and I believe it's fixing a recent regression.

SelectConversation is supposed to find an existing convo or create a new pending one, but it was always creating a new pending one, because it checks for non-null conversationIDKey in the `participants` flow to see whether a convo exists, and nothing was setting it.

So to repro, e.g. go to the profile page for someone you already have a chat with and click on Chat with them, and you'll get a new inbox row for a pending chat with them instead of being taken to the one that exists.